app/helpers/series_helper.rb

```diff
@@ -7,7 +7,11 @@ def show_series_data(work)
 
   # this should only show prev and next works visible to the current user
   def series_data_for_work(work)
-    series = work.series.select(&:visible?)
+    series = if work.preview_mode
+               work.series
```

Comment:
The `save_series` function seems to be designed to make this line of code work, but it might be simpler to change this line of code:
```suggestion
               work.serial_works.map(&:series).compact
```
The `serial_works` list is already being updated by the `series_attributes=` function, so we don't need extra code to update `work.series` if we read series info from `work.serial_works` instead.

(Alternatively, we could just add `work.new_series` to the list of series to be displayed.)